### PR TITLE
add new term MONDO:0100127 Imerslund-Grasbeck syndrome 2

### LINKED ISSUE #1536: split term: Imerslund-Grasbeck syndrome MONDO:0009853
Term to split: Imerslund-Grasbeck syndrome MONDO:0009853 

Names for new terms: Imerslund-Grasbeck syndrome 1, Imerslund-Grasbeck syndrome 2

List properties that should be moved to new term
OMIM:261100 needs to be moved to Imerslund-Grasbeck syndrome 1

OMIM has created IMERSLUND-GRASBECK SYNDROME 2 (https://omim.org/entry/618882) and renamed 261100 as IMERSLUND-GRASBECK SYNDROME 1; IGS1, with gene-specific definitions for each

I am not certain about some of the other terms now listed under https://www.omim.org/entry/261100, i.e. whether they are alternates for Imerslund-Grasbeck syndrome or Imerslund-Grasbeck syndrome 1.

### LINKED ISSUE #1802: split term:Imerslund-Grasbeck syndrome into: generic+type1
Term to split: Imerslund-Grasbeck syndrome

Name for new term:

aside: @nicolevasilevsky cna you update this templaate. With a split there aare 2+ new terms, not one

 - Imerslund-Grasbeck syndrome
 - Imerslund-Grasbeck syndrome type 1

List properties that should be moved to new term

We need documentation on this split pattern. ORDO tends to represent the generic. Most properties will move to the generic. Use judgment. Eg syns that end in "1" should move to subclass

```
name: imerslund-grasbeck syndrome
xref: OMIMPS:261100 {source="MONDO:equivalentTo"} ! imerslund-grasbeck syndrome
```

### PR DESCRIPTION
**Pull request by nicolevasilevsky**
closes #1536
closes #1802

### CODE REVIEW

File: src/ontology/mondo-edit.obo [hunk 1]
```diff
@@ -432996,6 +432997,20 @@ xref: ICD10:F16.183 {source="MONDO:relatedTo"}
 is_a: MONDO:0024417 {source="https://orcid.org/0000-0001-5208-3432"} ! perceptual disorders
 created_by: http://orcid.org/0000-0001-5208-3432
 
+[Term]
+id: MONDO:0100127
+name: Imerslund-Grasbeck syndrome 2
+def: "An autosomal recessive disorder characterized by onset of megaloblastic anemia associated with decreased serum vitamin B12 (cobalamin, Cbl) in infancy or early childhood. Low molecular weight (LMW) proteinuria is frequently present, but usually occurs later and is usually mild or subclinical. Patients often present with vague symptoms, including failure to thrive, loss of appetite, fatigue, lethargy, and/or recurrent infections." [OMIM:618882]
+synonym: "megaloblastic anemia, norwegian type" EXACT [OMIM:618882]
+xref: OMIM:618882 {source="MONDO:equivalentTo"}
+is_a: MONDO:0001700 {source="OMIM:618882"} ! megaloblastic anemia (disease)
```
Comment (cmungall) [inline src/ontology/mondo-edit.obo:433006]: hmm, feels like is-a overloaading and doesn't strictly come from omim. Seems like the isas for the generic type came from ordo, you coulld cite those here.. or review whether this is warraanted aat all, some of the parents may be in the obs candidates

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:433006]: I added the parent class Imerslund-Grasbeck syndrome and moved Type 1 and Type 2 as children.

The parent megaloblastic anemia comes from NCIT:C131677 and DC-OMIM:261100

Comment (nicolevasilevsky) [reply src/ontology/mondo-edit.obo:433006]: Let me know if further changes are needed

**Comment by nicolevasilevsky** [comment]
I need to redo this, to properly follow the intended workflow:
https://mondo.readthedocs.io/en/latest/editors-guide/splitting-classes/

**Pull request closed without merging**